concent_api/core/tests/utils.py

```diff
@@ -954,4 +954,4 @@ def _get_verification_deadline_as_timestamp(
 
     @staticmethod
     def _get_uuid() -> str:
-        return str(uuid.uuid4())
+        return generate_uuid()
```

Comment:
I don't understand this method. Name _get... suggests me, you are asking for a particular field. For field, that is used multiple times (e.g. some Id or key, the same used a few times). This name is mistakenly for me, it suggest that you someway prepared Id and now get access to this particular Id. 

Please remove `_get_uuid()` and use `generate_uuid()` instead in all places. Unless you have good reason to keep it :)

Reply:
The reason for keeping it inside this class is that you can use this method directly in tests without a need to import `generate_uuid` in every file. 
About the name, we use `_get_something` all around in the tests to retrieve something, ie. `_get_serialized_force_payment` etc.

Reply:
If you say so... I don't think it is good approach. Please change name to `_get_random_uuid`(would recommend) or leave as it is now

Reply:
Please check out this scenario;
Currently we have a `generate_uuid` function and test utils method `_get_uuid` for those tests which inherits from `ConcentIntegrationTestCase` class. Also, currently `_get_uuid` only calls `generate_uuid`. However, we can't be sure that in the future we will not be changing something or doing something extra in `_get_uuid`. With the current way we will not have to update anything in all tests if something changes only in this method.

About `_get_random_uuid` it is no longer random, but depends on the `last_char` passed (for the same `last_char` there will be the same returned value).